src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:29589/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:29589/project/issues?id=sample&branch=feature2&issues=AWDHLOLOCX23U5nE1m08&open=AWDHLOLOCX23U5nE1m08)

[ISSUE_KEY]: AWDHLOLOCX23U5nE1m08